lib/index.js

```diff
@@ -48,6 +48,36 @@ function remove(tracks) {
     return TrackPlayer.remove(tracks);
 }
 
+function registerEventHandler(handler) {
+    if (Platform.OS !== 'android') {
+        function warpResponse(event, payload) {
+            // transform into headlessTask format and return to handler
+            const additionalKeys = payload || {};
+            handler({ type: event, ...additionalKeys });
+        }
+
+        const events = [
+            'playback-state',
+            'playback-error',
+            'playback-progress',
+            'playback-queue-ended',
+            'playback-track-changed',
+
+            'remote-play',
+            'remote-next',
+            'remote-stop',
+            'remote-pause',
+            'remote-previous',
+        ]
+
+        for (let i = 0; i < events.length; i++) {
+            eventEmitter.addListener(event, (payload) => { warpResponse(event, payload) });
```

Comment:
`event` does not exist?

I think you can replace `(payload) => { warpResponse(event, payload) }` with `wrapResponse.bind(null, events[i])`